embedded-postgresql/src/main/java/com/playtika/test/postgresql/EmbeddedPostgreSQLBootstrapConfiguration.java

```diff
@@ -94,4 +100,14 @@ private void registerPostgresqlEnvironment(GenericContainer postgresql,
         MapPropertySource propertySource = new MapPropertySource("embeddedPostgreInfo", map);
         environment.getPropertySources().addFirst(propertySource);
     }
+    
+	private void putRequiredFileOnContainer(GenericContainer postgresql,
+			FileRequiredProperties fileRequiredProperties) {
+		fileRequiredProperties.getPathForFiles().stream()
+				.filter(Objects::nonNull)
+				.filter(fileDetails -> !StringUtils.isEmpty(fileDetails.getCopyPath())
```

Comment:
instead of checking here for `isEmpty` please add validation into configuration. you can use existing configurations as example. Also please don't forget to put `@Validated` over configurationProperties class